김진/231206/K번째수.js

```diff
@@ -0,0 +1,20 @@
+function solution(array, commands) {
+  let answer = [];
+  const result = [];
+  for (let i = 0; i < commands.length; i++) {
+    answer = array.slice(commands[i][0] - 1, commands[i][1]).sort();
```

Comment:
```suggestion
  const result = [];
  for (let i = 0; i < commands.length; i++) {
    const answer = array.slice(commands[i][0] - 1, commands[i][1]).sort();
```
answer를 따로 값을 바꾸는게 없으니까 이렇게 하는게 더 좋을 것 같아요.